workflow/types.go

```diff
@@ -29,3 +31,13 @@ type Action struct {
 	Volumes     []string          `yaml:"volumes,omitempty"`
 	Environment map[string]string `yaml:"environment,omitempty"`
 }
+
+// Template represents a workflow template
+type Template struct {
+	ID        string    `json:"id,omitempty"`
+	Name      string    `json:"name,omitempty"`
+	CreatedAt time.Time `json:"created_at,omitempty"`
+	UpdatedAt time.Time `json:"updated_at,omitempty"`
+	DeletedAt time.Time `json:"deleted_at,omitempty"`
+	Data      string    `json:"data,omitempty"`
```

Comment:
Technically speaking we have a Workflow type in `workflow/types.go`, I think `Data` should be that type, can you double check? Thanks